### ISSUE TITLE
chromebook dpad/stick issues

### ISSUE THREAD

**Issue by Grumm1291** [nbarkhina/N64Wasm#8]
I am currently playing on a chromebook and am trying to play a game called the new tetris, but for some reason it just cant read the stick and dpad inputs (im using a keyboard, i use arrow keys for dapad and nothing happens same for cstick controls), all the other button controls work, just not dpad or stick

**Comment by nbarkhina** [comment]
Yes the D-pad on keyboard currently does not work, it is on my list of things to fix. The stick buttons should work though, if you mapped your arrow keys to the up/down/left/right that translates to stick movements.

**Comment by nbarkhina** [comment]
I added the D-pad controls, you can try it here

https://neilb.net/n64wasm/v2/
